datasetinsights/data/datasets/__init__.py

```diff
@@ -6,11 +6,11 @@
 from .synthetic import SynDetection2D
 
 __all__ = [
-    Cityscapes,
-    CocoDetection,
-    Dataset,
-    GroceriesReal,
-    GoogleGroceriesReal,
-    NyuDepth,
-    SynDetection2D,
+    "Cityscapes",
```

Comment:
https://docs.python.org/3/reference/simple_stmts.html#the-import-statement

Reply:
The public names defined by a module are determined by checking the module’s namespace for a variable named __all__; if defined, it must be a sequence of strings which are names defined or imported by that module. The names given in __all__ are all considered public and are required to exist. If __all__ is not defined, the set of public names includes all names found in the module’s namespace which do not begin with an underscore character ('_'). __all__ should contain the entire public API. It is intended to avoid accidentally exporting items that are not part of the API (such as library modules which were imported and used within the module).

Reply:
I think it's fair to assume all of these module are public api. The only reason to hav __all__ is to prevent flake8 F401 [error](https://flake8.pycqa.org/en/2.6.0/warnings.html). We might as well remove __all__ statement and add #noqa string here.

Reply:
Do you want me to add the #noqa in this PR? 
Also not sure what #noqa is.

Reply:
https://flake8.pycqa.org/en/3.1.1/user/ignoring-errors.html#in-line-ignoring-errors

Reply:
https://docs.python.org/3/tutorial/modules.html#importing-from-a-package